etna/models/prophet.py

```diff
@@ -152,6 +154,76 @@ def predict(self, df: pd.DataFrame, prediction_interval: bool, quantiles: Sequen
         y_pred = y_pred.rename(rename_dict, axis=1)
         return y_pred
 
+    def _check_mul_components(self, ignore_components: Set[str]):
+        """Raise error if model contains multiplicative components."""
+        components_modes = self.model.component_modes
+        if components_modes is None:
+            raise ValueError("This model is not fitted!")
+
+        mul_components = set(self.model.component_modes["multiplicative"])
+        if len(mul_components - ignore_components) > 0:
```

Comment:
Looks strange, can't we just check `len(mul_components) > 0`?

Reply:
Here, `mul_components` will contain `multiplicative_terms` in any case and may contain `extra_regressors_multiplicative` if regressors are fitted. So we need to filter this out to obtain names of the multiplicative components. the model.